core/01_core.js

```diff
@@ -314,7 +314,6 @@
       error,
     ) => ops.op_abort_wasm_streaming(rid, error),
     destructureError: (error) => ops.op_destructure_error(error),
-    terminate: (exception) => ops.op_terminate(exception),
```

Comment:
Since this was being renamed to `op_dispatch_exception`, I took the chance to just remove it and use the op directly